tox.ini

```diff
@@ -1,16 +1,11 @@
 [tox]
-envlist = py{27,36}, coverage, flake8
+envlist = py{27,36}, flake8
 
 [testenv]
-passenv = DISPLAY
-deps = -rrequirements/tests.txt
-commands = pytest --driver Firefox {posargs}
-
-[testenv:coverage]
 passenv = DISPLAY TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
 deps = -rrequirements/coverage.txt
 commands =
-    pytest --driver Firefox --cov {posargs}
+    pytest --driver=Firefox --cov --html=results/{envname}.html {posargs}
```

Comment:
The coveralls results should be merged so that we reach 100% coverage due to the different code-paths each version may take. You shouldn't get any emails from coveralls (I don't) and there should be just one comment on the pull request per push, which might trigger an email notification depending on your GitHub settings.